routes/web.php

```diff
@@ -28,8 +28,11 @@
 	->only(['index', 'store', 'edit', 'update', 'destroy'])
 	->middleware(['auth', 'verified']);
 
+Route::post('/comments/{comment}', [CommentController::class, 'destroy'])->middleware(['auth', 'verified'])->name('comments.destroy');
+Route::get('/comments/{chirp}', [CommentController::class, 'index'])->name('comments.index');
+
 Route::resource('comments', CommentController::class)
-    ->only(['store', 'update', 'destroy'])
+    ->only(['store', 'update', 'destroy', 'index'])
```

Comment:
I'm not a big fan of the resource method, it hides the routes from me when I would rather just see each route

Reply:
I totally agree